src/app/products/products.component.html

```diff
@@ -0,0 +1,99 @@
+<div class="container">
+
+  <mat-card>
+
+    <div fxLayout="row" fxLayout.lt-md="column">
+
+      <div fxFlex="50%">
+
+        <mat-nav-list>
+
+          <mat-list-item [routerLink]="['']">
```

Comment:
Remove these blank routerLinks from the list items. They are not required right now.